_layouts/partners/state-and-local.html

```diff
@@ -1,95 +1,98 @@
-                <div class="tablet:grid-col-12 partners-subtitle"> {{ page.body | markdownify }}</div>
-            </div>
-            <div class="grid-row">
-                <div class="tablet:grid-col-12">{{ page.partnership_steps_title | markdownify }}</div>
-            </div>
-            <div class="grid-row">
-                <div class="tablet:grid-col-12 partners-body">
-                    <ol class="usa-process-list usa-process-list--big usa-process-list--connected">
-                        <li class="usa-process-list__item">
-                            <h3 class="usa-process-list__heading">{{ page.partnership_steps_header1 }}</h3>
-                            <div class="margin-top-05">
-                                {{ page.partnership_steps_body1 }}
-                                <a href="{{ site.baseurl }}/partners/contact/" class="caret">Contact an account manager to get started</a>
-                            </div>
-                        </li>
-                        <li class="usa-process-list__item">
-                            <h3 class="usa-process-list__heading">{{ page.partnership_steps_header2 }}</h3>
-                            {{ page.partnership_steps_body2 | markdownify }}
-                        </li>
-                        <li class="usa-process-list__item">
-                            <h3 class="usa-process-list__heading">{{ page.partnership_steps_header3 }}</h3>
-                            {{ page.partnership_steps_body3 | markdownify }}
-                        </li>
-                        <li class="usa-process-list__item">
-                            <h3 class="usa-process-list__heading">{{ page.partnership_steps_header4 }}</h3>
-                            {{ page.partnership_steps_body4 | markdownify }}
-                        </li>
-                    </ol>
-                </div>
-            </div>
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <h1 class="partners-title">{{ page.title }}</h1>
+        <div class="tablet:grid-col-12 partners-border"></div>
+        <div class="tablet:grid-col-12 partners-subtitle">{{ page.body | markdownify }}</div>
+      </div>
+      <div class="grid-row">
+        <div class="tablet:grid-col-12">{{ page.partnership_steps_title | markdownify }}</div>
+      </div>
+      <div class="grid-row">
+        <div class="tablet:grid-col-12 partners-body">
+          <ol class="usa-process-list usa-process-list--big usa-process-list--connected">
+            <li class="usa-process-list__item">
+              <h3 class="usa-process-list__heading">{{ page.partnership_steps_header1 }}</h3>
+              <div class="margin-top-05">
+                {{ page.partnership_steps_body1 }}
+                <a href="{{ site.baseurl }}/partners/contact/" class="caret"
+                  >Contact an account manager to get started</a
+                >
```

Comment:
Is this typical formatting, or should the whole close tag be on this line?

Reply:
I'll put both close tags on the same line, cleaner that way